plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -132,9 +133,9 @@ public IcebergFileWriter createDataFileWriter(
         switch (fileFormat) {
             case PARQUET:
                 // TODO use metricsConfig
-                return createParquetWriter(outputPath, icebergSchema, jobConf, session, hdfsContext);
+                return createParquetWriter(MetricsConfig.getDefault(), outputPath, icebergSchema, jobConf, session, hdfsContext);
```

Comment:
Is there a GH Issue to support metrics mode for Parquet? If not, can you make one

Either way, link it here